package/libs/libtraceevent/Makefile

```diff
@@ -1,12 +1,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libtraceevent
-PKG_VERSION:=1.6.1
+PKG_VERSION:=1.6.2
```

Comment:
Please squash the version updates into the initial versions. As you are anyway adding these as new libraries or jumping over many versions no need for separate commits.